rdm/checklists/62304_2006-05_class_b.txt

```diff
@@ -0,0 +1,9 @@
+# Audit checklist for IEC62304 version 2006-05 Class B products
+include 62304_2006-05_class_a.txt
```

Comment:
These checklists look really great! Thanks for putting them together, I bet it was a lot of work!

Reply:
When I have a chance I'll read through some of them in more detail.